.github/workflows/tests.yml

```diff
@@ -0,0 +1,102 @@
+---
+name: Terraform modules tests
+
+on:
+  schedule:
+    - cron: '0 1 * * 2'
```

Comment:
it's about a scheduled job. Renovate isn't impacted by that since `pull_request` will run regardless.
Nevertheless, can't hurt to put it to Monday as sanity check that whatever was auto-merged is working all-together.

Reply:
I'm fine keeping that on Tuesday.
Assumption: Renovate runs on the weekend. On Monday we merge the PRs / check / adjust depending on wh at Renovate says.
Tuesday kicks-in as sanity check to whatever was done on Monday.

Maybe we figure out later that we want to run it more often, we'll see.